docs/reference/commandline/cli.md

```diff
@@ -78,6 +78,11 @@ For example:
 Instructs Docker to use the configuration files in your `~/testconfigs/`
 directory when running the `ps` command.
 
+> **Note:**
+> Calling docker with `sudo` results in Docker looking first in `/root/.docker/`.
+> If it isn't found it looks for the user who did the `sudo` call and resolves the
```

Comment:
Yes, I doubted about that one; OTOH, wondering if most users will consider "the user that did the sudo call" the "current" user? i.e. if I'm logged in as "sebastiaan", and run `sudo foo`, I can imagine most people will consider "sebastiaan" the current user. But open to suggestions